src/vivarium_gates_mncnh/components/lbwsg.py

```diff
@@ -414,3 +414,23 @@ def calculate_paf(self, x: pd.DataFrame) -> float:
         paf = (mean_rr - 1) / mean_rr
 
         return paf
+
+    def _load_custom_paf_exposure(self, builder: Builder) -> pd.DataFrame:
+        """Loads custom exposure data for the PAF simulation. This swaps the birth
+        exposure with the early neonatal exposure for the early neonatal age group.
+        TODO: potentially switch ENN for LNN exposure depending on how this validates.
+        """
+        birth_exposure = builder.data.load(data_keys.LBWSG.BIRTH_EXPOSURE)
+        neonatal_exposure = builder.data.load(data_keys.LBWSG.EXPOSURE)
+        col_order = neonatal_exposure.columns.tolist()
+        enn_exposure = neonatal_exposure.loc[neonatal_exposure["child_age_end"] < 8 / 365.0]
+        enn_exposure = enn_exposure.drop(columns=["value"])
+        new_enn_exposure = pd.merge(
+            birth_exposure,
+            enn_exposure,
+            on=["sex_of_child", "year_start", "year_end", "parameter"],
+        )
+        exposure = neonatal_exposure.loc[neonatal_exposure["child_age_start"] > 6 / 365.0]
```

Comment:
Just double checking that 8 / 365 and 6 / 365 are the right numbers.

Reply:
Yes Im being explicit by being less than or greater than the age bin edge so we don't have to worry if it is inclusive or not.